docs/src/manifolds/generalunitary.md

```diff
@@ -7,21 +7,22 @@ Both [`OrthogonalMatrices`](@ref) and [`UnitaryMatrices`](@ref) are quite simila
 ```@autodocs
 Modules = [Manifolds]
 Pages = ["manifolds/Orthogonal.jl"]
-Order = [:type, :function]
+Order = [:type, :function, :constant]
 ```
 
 ## Unitary Matrices
 
 ```@autodocs
 Modules = [Manifolds]
 Pages = ["manifolds/Unitary.jl"]
-Order = [:type, :function]
+Order = [:type, :function, :constant]
 ```
 
+
 ## [Common functions](@id generalunitarymatrices)
 
 ```@autodocs
 Modules = [Manifolds]
 Pages = ["manifolds/GeneralUnitaryMatrices.jl"]
-Order = [:type, :function]
+Order = [:type, :function, :constant]
```

Comment:
Since the constants here are the Lie groups, we could put those first to always first habe the Lie group definition?

Reply:
Sure, I'll change that.

Reply:
Should I change all `:constant` to be the first in order in groups.md file?

Reply:
hehe, See my comment below, yes please.